ergotree-ir/src/sigma_protocol/dlog_group.rs

```diff
@@ -85,6 +85,26 @@ pub fn exponentiate(base: &EcPoint, exponent: &Scalar) -> EcPoint {
     }
 }
 
+/// Attempt to decode
+pub fn from_bytes(bytes: impl AsRef<[u8]>) -> Result<EcPoint, SerializationError> {
```

Comment:
Do you have a reason to have parsing code outside of `SigmaSerializable` implementation besides test for `eval`? 
`EcPoint::parse_from_bytes(bytes)` will give the same result as this `from_bytes`.